c2FmZQ/internal/crypto/passphrase.go

```diff
@@ -13,7 +13,10 @@ import (
 // Passphrase retrieves a passphrase. If cmd is set, the passphrase is the
 // output of the command. Or, if file is set, the passphrase is the content
 // of the file. Otherwise, the passphrase is read from the terminal.
-func Passphrase(cmd, file string) ([]byte, error) {
+func Passphrase(cmd, file, env string) ([]byte, error) {
+	if env != "" {
+		return []byte(env), nil
```

Comment:
Shouldn't this be `[]byte(os.Getenv(env))` ?

Otherwise, the passphrase will be the value of the flag.